ingestion/src/metadata/cli/backup.py

```diff
@@ -143,5 +186,11 @@ def run_backup(  # pylint: disable=too-many-arguments
     )
 
     if upload:
-        endpoint, bucket, key = upload
-        upload_backup(endpoint, bucket, key, out)
+        if uploadDestinationType == "AWS":
```

Comment:
Here we could have an Enum with `AWS` and `Azure` as possible options to make it easier to know what scenarios we are covering

Reply:
Added Enum class in utils folder. Not sure if this is the appropriate location. Please could you review?

Reply:
Thanks for the changes @amymareemc!

A couple of notes here:
1. The upload destination type only impacts the backup module. We can create the Enum on `backup.py` directly after the imports.
2. Let's try to follow PEP8 naming conventions (https://peps.python.org/pep-0008/). Classes can be named in CamelCase, e.g., `UploadDestinationType`
3. This is just a personal preference, but I like Enums that express written constants to be strings, E.g.:

```python
class UploadDestinationType(Enum):
    AZURE = "Azure"

# Then you can compare as
if uploadDestinationType == UploadDestionationType.AZURE.value
```

This allows for string comparison of values without hardcoding anything

Reply:
Thanks so much for the notes, implementing these changes now!
Just on point 1, the Enum type is also used in cmd.py (see[ line 286](https://github.com/open-metadata/OpenMetadata/pull/8334/commits/084e6f8ebb431c0dfaf7e6197c7a853bec6551e9#diff-2b38722110cfd94ca38c333ca42feb6387e283888b5dda34081e7421c83fe925L286)). Given this, should it still just be created in `backup.py`?

Reply:
yess that should not be an issue, `cmd.py` can import the enum from `backup.py` directly